### ISSUE TITLE
Could the vertical_full layout replace the single one?

### ISSUE THREAD

**Issue by denizonm** [Vidnez/Drastic-Patch-for-RGB30-20sx#2]
The "full" layout is just a better version of the "single" layout in my opinion since both uses the entire horizontal screen but also shows a little preview of the other ds screen.

I don't know anything about how to compile it but couldn't it be done by copying the values of 
```
	[DISP_TGT_MODE_TOP_FULL] = {
		.tgt_rect = {
			{0, 0, 720, 540},
			{240, 540, 240, 180},
		},
	},
```
to
```
	[DISP_TGT_MODE_H_SINGLE] = {
		.tgt_rect = {
			{0, 90, 720, 540},
			{0, 0, 0, 0},
		},
	},
```
and replacing the bg_horizon_single.png with the bg_vertical_full.png one?

Thank you.

**Comment by Vidnez** [comment]
Yes, both designs seem redundant, but replacing one with the other eliminates a layout and there are people who prefer to have a single centered screen and swap screens with a button.

**Comment by denizonm** [comment]
That makes sense. Could you tell me how I can compile my own version for my personal use?

thank you
